File: android/app/src/main/java/com/wangzhenman/notifysentinel/model/Device.kt
package com.wangzhenman.notifysentinel.model

import com.google.gson.annotations.SerializedName

data class Device(
    val id: Long,
    val name: String,
    val platform: String,
    val token: String,
    val enabled: Boolean,
    @SerializedName("created_at")
    val createdAt: String? = null,
    @SerializedName("updated_at")
    val updatedAt: String? = null,
)

File: server/models/device.go
package models

import "time"


type Device struct {

	ID uint `gorm:"primaryKey" json:"id"`

	// 设备名称，例如：小米14
	Name string `json:"name"`

	// 平台，例如：
	// mipush
	// fcm
	// apns
	Platform string `json:"platform"`


	// 推送Token
	Token string `json:"token"`


	// 是否启用
	Enabled bool `json:"enabled"`


	CreatedAt time.Time `json:"created_at"`

	UpdatedAt time.Time `json:"updated_at"`
}